### DISCUSSION TITLE
External Links

### DISCUSSION THREAD

**Discussion by mikescola** [buildnatively/community#6] [Ideas]
### Please confirm you've checked the following resources:

- [X] I've searched through the [Docs](https://docs.buildnatively.com).
- [X] I've checked [Natively Youtube Playlist 1](https://youtube.com/playlist?list=PLYo9EJqlVMNdtzb_rm6_6kjlDCmaoAqlO&si=sOygZtRDGUn_2Yi8) and [Natively Youtube Playlist 2](https://youtube.com/playlist?list=PLYo9EJqlVMNcwz2o1Xx5xVYYzzKgcyMeB&si=f4GuCPvUAdjvHplD).
- [X] I've reviewed the [Troubleshooting Guides](https://docs.buildnatively.com/guides/troubleshooting).
- [X] I've visited the [FAQ](https://docs.buildnatively.com/natively-platform/faq).

### Which web app platform are you using?

Other

### Which SDK are you using?

JavaScript SDK

### JS SDK / Bubble Plugin Version (e.g., 2.12.1)

2.12.1

### Platform with the issue?

Android

### Steps to reproduce

1. Execute an external link: window.natively.openExternalURL('https://www.google.com', true); (or false)

### Expected results

Open a link!

### Actual results

Nothing happens. There are also no errors thrown.

**Comment by romanfurman6** [comment]
Hello @mikescola try to upgrade the plugin to the latest version 2.12.2 + rebuild your app it should fix the issue 🙌

**Reply by mikescola** [reply]
I updated to 2.12.2 and did an app rebuild and it is still not working. No output or errors either.

**Reply by romanfurman6** [reply]
can you share how do you use that open external URL method ? like in docs ? https://docs.buildnatively.com/guides/integration/open-an-external-app-url#javascript-sdk

**Reply by mikescola** [reply]
I'm using it exactly like in the docs! I'm passing a true boolean for what it's worth.

**Reply by romanfurman6** [reply]
hmm i think the problem is in your build SDK version, can you pls send it
https://docs.buildnatively.com/guides/integration/device#javascript-sdk
`sdkVersion`

**Reply by mikescola** [reply]
It says SDK Version 19. I'm using the latest 2.12.2.

**Reply by romanfurman6** [reply]
I think i found an issue, give me few minutes

**Reply by romanfurman6** [reply]
can you pls try to rebuild, should be fixed now

**Reply by romanfurman6** [reply]
sorry for this kind of issue, very tricky bug 🤯

**Reply by mikescola** [reply]
Fixed! Amazing! Thank you for you hard work.
